collect/queue.go

```diff
@@ -56,9 +56,21 @@ func Flush() {
 }
 
 func send() {
+
+	lastSent := time.Now()
+
 	for {
 		qlock.Lock()
 		if i := len(queue); i > 0 {
+
+			// If there is no backlog, wait for the minimum send interval before sending any more data
```

Comment:
@captncraig 
We are using a larger batch size, expecting to not normally have a backlog, and we want to limit the frequency at which we send these larger batches. In the scenario where there is a backlog, we don't want to limit the sending frequency, to avoid the backlog growing.

What do you mean regarding throwing away a partial batch? We don't intend to discard anything, if you have spotted something then please let us know.